docs/conf.py

```diff
@@ -44,11 +44,17 @@
         ]
     )
 
-docs_core = ROCmDocs("hipfort Documentation")
+getVersionCmd = r'sed -n -e "s/^.*HIPFORT_VERSION.* \"\([0-9\.]\{1,\}\).*/\1/p" ../CMakeLists.txt'
+version = subprocess.getoutput(getVersionCmd)
```

Comment:
How about something directly in Python? If nothing else, it makes it easier to do error checking.

```python
import re

with open('../CMakeLists.txt', encoding='utf-8') as f:
    match = re.search(r'.*\bHIPFORT_VERSION\s+\"?([0-9.]+)[^0-9.]+', f.read())
    if not match:
        raise ValueError("HIPFORT_VERSION not found!")
    version = match[0]
```

Reply:
Sounds good to me. Done in [use regex for getting hipfort version number](https://github.com/ROCmSoftwarePlatform/hipfort/pull/113/commits/cf4ba174a61bbbafa11ea6ffa81265559cbf794d)